scripts/start-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# The current directory is a mounted volume, and is owned by the
+# user running Docker on the host machine.
+#
+# We don't want to trample all over the contents of this directory
+# with files owned by root. So create a new user with the same UID,
+# and drop privileges before running any commands.
+
+# Get the numeric user ID of the current directory.
+uid=$(ls -nd . | awk '{ print $3 }')
+
+# Create an `olympia` user with that ID, and the current directory
+# as its home directory.
+useradd -Md $(pwd) -u $uid olympia
```

Comment:
Nope. There's a bug that's several years old, but there's no real solution. At first I tried just creating the user from the Dockerfile, but there's no way to give it the same UID as the user doing the build, and I didn't want people to have to edit the Dockerfile just to add their UID. Plus, hard coding that information in the image seems like a bad idea.